astroplan/core.py

```diff
@@ -359,6 +415,22 @@ def altaz(self, time, target=None, obswl=None):
             If ``target`` is `None`, returns `~astropy.coordinates.AltAz` frame.
             If ``target`` is not `None`, returns the ``target`` transformed to
             the `~astropy.coordinates.AltAz` frame.
+
+        Examples
+        --------
+        Create an instance of the `~astropy.coordinates.AltAz` frame for an
+        observer at Apache Point Observatory at a particular time; and
+        transform that
+
+        >>> from astroplan import Observer
+        >>> from astropy.time import Time
+        >>> from astropy.coordinates import SkyCoord
+        >>> apo = Observer.at_site("APO")
+        >>> time = Time('2001-02-03 04:05:06')
+        >>> target = SkyCoord(0*u.deg, 0*u.deg)
+        >>> altaz_frame = apo.altaz(time)
+        >>> target_transformed_to_altaz_frame = target.transform_to(altaz_frame)
+        >>> target_transformed_to_altaz_frame = apo.altaz(time, target) # Alternatively
```

Comment:
I avoided showing the output because the `repr`s may be in flux (#53, astropy/astropy#4055). If/when those are merged/closed and the `repr`s won't lead to more confusion, we can show the outputs. 

At least once, I went through some gymnastics to avoid showing `repr`s while still showing outputs, like [here, for example](https://github.com/astroplanners/astroplan/pull/59/files#diff-8237cb1c7bdb65945cb3e8f60519cf85R1473). Would you like to see that kind of print statement in this alt/az example as well?